src/RTT_Format_Reader/CellDefs.cc

```diff
@@ -84,75 +79,67 @@ void CellDefs::readEndKeyword(ifstream & meshfile)
  *        definitions.
  */
 void CellDefs::redefineCellDefs(
-    vector_vector_int const & cell_side_types,
-    std::vector< std::vector< std::vector< size_t > > >
-    const & cell_ordered_sides)
-{
-    Insist(cell_side_types.size()
-           == static_cast<size_t>(dims.get_ncell_defs()),
-	   "Error in supplied cell redefinition side types data.");
-    Insist(cell_ordered_sides.size()
-           == static_cast<size_t>(dims.get_ncell_defs()),
-	   "Error in supplied cell redefinition ordered side data.");
+    vector_vector_int const &cell_side_types,
+    std::vector<std::vector<std::vector<size_t>>> const &cell_ordered_sides) {
+  Insist(cell_side_types.size() == static_cast<size_t>(dims.get_ncell_defs()),
+         "Error in supplied cell redefinition side types data.");
+  Insist(cell_ordered_sides.size() ==
+             static_cast<size_t>(dims.get_ncell_defs()),
```

Comment:
I can see why the continuation matches opening parenthesis but why does the equality operator cause the next line to be so far to the right?